lessons/24-js-loops/index.js

```diff
@@ -0,0 +1,109 @@
+// Add all the numbers between 0 and 1000 using for statement
+let sum=0;
+for (let i=1;i<=1000; i++)
+{
+    sum=sum+i; 
+    if (sum > 400)
+    {break;}
+    console.log(sum);
+}
+// Show numbers from 0 to 10 using For loop
+for (j=0;j<=10;j++)
+{
+    console.log(j);
+}
+// Show numbers from 100 to 0 using For loop
+for (k=100;k>=0;k--)
+{
+    console.log(k);
+}
+// Show even numbers between 0 and 100 using For loop
+for (l=0;l<100;l=l+2)
+{
+    let l1=l+2;
+    if (l1>=100)
+    {break;}
+    else {
+        console.log(l1);
+    }
+}
+// Produce the following output using For loop
+let star = ("*");
+for (m=1;m<=13;m++)
+{
+    for (n=1;n<=m;n++);
+    {
+        console.log(star);
+        star = star + "*";
+    }
+}
+// Show numbers from 0 to 10 using while loop
+let a=0;
+while (a<=10)
+{   console.log(a);
+    a++;
+}
+// Show numbers from 100 to 0 using while loop
+let b=100;
+while (b>=0)
+{
+    console.log(b);
+    b--;
+}
+// Show even numbers between 0 and 100 using while loop
+let c=0;
+while (c<100)
+{
+    c=c+2;
+    if (c==100) {break;}
```

Comment:
The `if` here isnt really needed since the while loop will terminate once `c >= 100`